pkgs/by-name/fr/freeimage/package.nix

```
@@ -14,17 +14,16 @@
   jxrlib,
   pkg-config,
   fixDarwinDylibNames,
-  autoSignDarwinBinariesHook,
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "freeimage";
-  version = "unstable-2021-11-01";
+  version = "unstable-2024-04-18";
```

Per [pkgs/README.md chapter Versioning](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning)

```suggestion
  version = "3.18.0-unstable-2024-04-18";
```

---

Thanks!